Broken Pipeline on project

**Describe the bug**
scheduled pipelines are failing due to a mssql install error during the mssql integration test

https://github.com/rwnx/pynonymizer/actions/runs/3667058815

We need to fix this. I've done some investigation and I think this is because the version of ubuntu we're using "ubuntu-latest" from github actions is not supported by the install/ apt sources we're using. We should lock it to a previous version (presumably the previous LTS? for now)

```
2023-03-26T02:35:21.9657157Z ##[group]Run actions/setup-python@v2
2023-03-26T02:35:21.9657417Z with:
2023-03-26T02:35:21.9657609Z   python-version: 3.8
2023-03-26T02:35:21.9658028Z   token: ***
2023-03-26T02:35:21.9658219Z env:
2023-03-26T02:35:21.9658412Z   MSSQL_SA_PASSWORD: pASSw0rd
2023-03-26T02:35:21.9658653Z   PYNONYMIZER_DB_USER: sa
2023-03-26T02:35:21.9658913Z   PYNONYMIZER_DB_PASSWORD: pASSw0rd
2023-03-26T02:35:21.9659128Z ##[endgroup]
2023-03-26T02:35:22.1208892Z Successfully setup CPython (3.8.16)
2023-03-26T02:35:22.1348852Z ##[group]Run bash tests_integration/mssql/install_mssql.sh
2023-03-26T02:35:22.1349250Z [36;1mbash tests_integration/mssql/install_mssql.sh[0m
2023-03-26T02:35:22.1406500Z shell: /usr/bin/bash -e {0}
2023-03-26T02:35:22.1406740Z env:
2023-03-26T02:35:22.1406965Z   MSSQL_SA_PASSWORD: pASSw0rd
2023-03-26T02:35:22.1407207Z   PYNONYMIZER_DB_USER: sa
2023-03-26T02:35:22.1407454Z   PYNONYMIZER_DB_PASSWORD: pASSw0rd
2023-03-26T02:35:22.1407746Z   pythonLocation: /opt/hostedtoolcache/Python/3.8.16/x64
2023-03-26T02:35:22.1408074Z   LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.8.16/x64/lib
2023-03-26T02:35:22.1408341Z ##[endgroup]
2023-03-26T02:35:22.1589760Z Adding Microsoft repositories...
2023-03-26T02:35:22.1738824Z   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
2023-03-26T02:35:22.1743883Z                                  Dload  Upload   Total   Spent    Left  Speed
2023-03-26T02:35:22.1744304Z 
2023-03-26T02:35:22.2417245Z   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0Warning: apt-key is deprecated. Manage keyring files in trusted.gpg.d instead (see apt-key(8)).
2023-03-26T02:35:22.2731126Z 
2023-03-26T02:35:22.2734355Z 100   983  100   983    0     0   9963      0 --:--:-- --:--:-- --:--:-- 10030
2023-03-26T02:35:22.4536243Z OK
2023-03-26T02:35:22.4694274Z   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
2023-03-26T02:35:22.4694958Z                                  Dload  Upload   Total   Spent    Left  Speed
2023-03-26T02:35:22.4695756Z 
2023-03-26T02:35:22.6996461Z   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
2023-03-26T02:35:22.7008872Z 100    14  100    14    0     0     60      0 --:--:-- --:--:-- --:--:--    60
2023-03-26T02:35:23.8720556Z Unable to handle repository shortcut '404: Not Found'
2023-03-26T02:35:23.8970903Z ##[error]Process completed with exit code 1.
```

--- a/.github/workflows/autoblack-pr.yml
+++ b/.github/workflows/autoblack-pr.yml
@@ -5,14 +5,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
-      - name: Set up Python 3.7
-        uses: actions/setup-python@v1
+      - uses: actions/setup-python@v1
         with:
-          python-version: 3.7
+          python-version: 3.10
       - name: Install Black
-        run: pip install black
-      - name: Run black --check .
-        run: black --check .
+        run: pip install "black~=22.6.0"
+      - run: black --check .
       - name: If needed, commit black changes to the pull request
         if: failure()
         run: |

--- a/.github/workflows/autoblack-pr.yml
+++ b/.github/workflows/autoblack-pr.yml
@@ -7,7 +7,7 @@ jobs:
       - uses: actions/checkout@v1
       - uses: actions/setup-python@v1
         with:
-          python-version: 3.10
+          python-version: "3.10"
       - name: Install Black
         run: pip install "black~=22.6.0"
       - run: black --check .

--- a/.github/workflows/pypi.yml
+++ b/.github/workflows/pypi.yml
@@ -32,7 +32,9 @@ jobs:
         run: pytest tests/
 
   integration-mssql: 
-    runs-on: ubuntu-latest
+    # The ubuntu version here needs to be synced with the setup instructions in /tests_integration/mssql/install_mssql.sh
+    # as different versions of mssql have different setup instructions https://learn.microsoft.com/en-us/sql/linux/quickstart-install-connect-ubuntu?view=sql-server-ver16
+    runs-on: ubuntu-20.04
     env:
       MSSQL_SA_PASSWORD: pASSw0rd
       PYNONYMIZER_DB_USER: sa

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
   -------------------------------------------------------------------
 ## [Unreleased]
+## [1.25.0] 2023-03-29
 ### Changed
 - Postgres FAKE_UPDATE strategy now uses an id-based randomization to pick from the seed table. 
   This speeds up updates on large tables, as the seed table no longer needs to be ordered for every row in the target table.

--- a/pynonymizer/__init__.py
+++ b/pynonymizer/__init__.py
@@ -1,3 +1,3 @@
-__version__ = "1.24.0"
+__version__ = "1.25.0"
 
 from pynonymizer.pynonymize import pynonymize as run

--- a/tests_integration/mssql/install_mssql.sh
+++ b/tests_integration/mssql/install_mssql.sh
@@ -1,5 +1,7 @@
 #!/bin/bash -e
 
+# Installs mssql-server-2019 on ubuntu 20.04
+
 set -e
 source /etc/lsb-release
 
@@ -7,8 +9,8 @@ MSSQL_PID='developer'
 
 echo Adding Microsoft repositories...
 sudo curl https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -
-sudo add-apt-repository "$(curl https://packages.microsoft.com/config/ubuntu/$DISTRIB_RELEASE/mssql-server-2019.list)"
-sudo add-apt-repository "$(curl https://packages.microsoft.com/config/ubuntu/$DISTRIB_RELEASE/prod.list)"
+sudo add-apt-repository "$(curl https://packages.microsoft.com/config/ubuntu/20.04/mssql-server-2019.list)"
+sudo add-apt-repository "$(curl https://packages.microsoft.com/config/ubuntu/20.04/prod.list)"
 
 echo Running apt-get update -y...
 sudo apt-get update -y

